File: java/src/test/java/team/codium/legacytraining/printdate/PrintDateTest.java
package team.codium.legacytraining.printdate;

import org.junit.jupiter.api.Test;
import team.codium.legacytraining.printdate.Calendar;
import team.codium.legacytraining.printdate.PrintDate;
import team.codium.legacytraining.printdate.Printer;


public class PrintDateTest {
	@Test
	public void printDate() throws Exception {
		PrintDate printDate = new PrintDate(new Calendar(), new Printer());

		printDate.printCurrentDate();

		// How can we test this function?
	}
}

File: go/printdate/printdate_test.go
package printdate_test

import (
	"printdate/printdate"
	"testing"
)

func Test_PrintDate(t *testing.T) {
	printDate := printdate.NewPrintDate(
		printdate.NewCalendar(),
		printdate.NewPrinter(),
	)

	printDate.PrintCurrentDate()

	// How can we test this function?
}
